tests/test_chaiverse/test_metrics.py

```diff
@@ -112,9 +112,11 @@ def test_get_raw_leaderboard(data_dir_mock, get_ids_mock, tmpdir):
 @vcr.use_cassette(os.path.join(RESOURCE_DIR, 'test_get_submission_metrics.yaml'))
 @freeze_time('2023-07-14 19:00:00')
 def test_get_leaderboard_row():
-    results = metrics.get_leaderboard_row(('wizard-vicuna-13b-bo4', {'meta-data-key': 'meta-data-value'}), developer_key="key")
+    results = metrics.get_leaderboard_row(('wizard-vicuna-13b-bo4', {'meta-data-key': 'meta-data-value', 'thumbs_up' : 10, 'thumbs_down': 10}), developer_key="key")
```

Comment:
hi, I think we should add a test to assert, if thumb_up or thumb_down has changed, get_subission_metrics will be called, and if not, it will not be called?

BTW, @alexandred to add any additional feedback on styles if any

Reply:
Thank you, I'll commit the tests required soon